modules/javafx.controls/src/main/java/javafx/scene/control/Skin.java

```diff
@@ -61,12 +72,31 @@
     public Node getNode();
 
     /**
-     * Called by a Skinnable when the Skin is replaced on the Skinnable. This method
-     * allows a Skin to implement any logic necessary to clean up itself after
-     * the Skin is no longer needed. It may be used to release native resources.
-     * The methods {@link #getSkinnable()} and {@link #getNode()}
-     * should return null following a call to dispose. Calling dispose twice
-     * has no effect.
+     * Called once when {@link Skin} is set.  This method is called after the previous skin,
+     * if any, has been uninstalled via its {@link #dispose()} method.
+     * The skin can now safely make changes to its associated control, like registering listeners,
+     * adding child nodes, and modifying properties and event handlers.
+     * <p>
+     * Application code must not call this method.
+     * <p>
+     * The default implementation of this method does nothing.
+     *
+     * @implNote
+     * Skins only need to implement {@code install} if they need to make direct changes to the control
+     * like overwriting properties or event handlers.  Such skins should ensure these changes are undone in
+     * their {@link #dispose()} method.
+     *
+     * @since 20
+     */
+    default public void install() { }
+
+    /**
+     * Called when a previously installed skin is about to be removed from its associated control.
```

Comment:
Use @code tag for `Skin` and `Control` in this description.

Reply:
I think, you accidentally marked this as resolved :)

Reply:
did I miss a requested change somewhere?

Reply:
In this sentence, `skin` and `control` are lower-case (so don't refer to a class) and don't refer to a specific `@param` argument, so I don't think they don't need code case.